components/nimbus/src/nimbus_client.rs

```diff
@@ -325,7 +326,15 @@ impl NimbusClient {
                     is_enrolled_set.insert(ee.slug.clone());
                     all_enrolled_set.insert(ee.slug.clone());
                 }
-                EnrollmentStatus::WasEnrolled { .. } => {
+                EnrollmentStatus::WasEnrolled { .. }
+                | EnrollmentStatus::Disqualified {
+                    reason: DisqualifiedReason::NotSelected,
+                    ..
+                }
+                | EnrollmentStatus::Disqualified {
+                    reason: DisqualifiedReason::NotTargeted,
+                    ..
+                } => {
```

Comment:
Do we need the `Disqualified` `NotSelected`/`NotTargeted` here? Or will `WasEnrolled` cover those cases?